scripts/zkai_attention_kv_d16_native_width_gate.py

```
@@ -0,0 +1,636 @@
+    if set(summary) != required:
+        raise AttentionKvD16NativeWidthGateError("receipt summary field drift")
+    if summary["route_id"] != route_id:
+        raise AttentionKvD16NativeWidthGateError("route id drift")
+    if summary["target_id"] != target_id:
+        raise AttentionKvD16NativeWidthGateError("target id drift")
+    if summary["proof_version"] != proof_version:
+        raise AttentionKvD16NativeWidthGateError("proof version drift")
+    if summary["required_backend_version"] != required_backend_version:
+        raise AttentionKvD16NativeWidthGateError("backend version drift")
+    if summary["key_width"] != key_width or summary["value_width"] != value_width:
+        raise AttentionKvD16NativeWidthGateError("width drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)